build.sbt

```diff
@@ -32,7 +32,7 @@ lazy val concept_api = (project in file("."))
     version := appProperties.value.getProperty("NDLAComponentVersion"),
     scalaVersion := Scalaversion,
     javacOptions ++= Seq("-source", "1.8", "-target", "1.8"),
-    scalacOptions := Seq("-target:jvm-1.8", "-unchecked", "-deprecation", "-feature"),
+    scalacOptions := Seq("-target:jvm-1.8", "-unchecked", "-deprecation", "-feature", "-Ypartial-unification"),
```

Comment:
I Scala 2.13 trenger man ikke dette lenger, er det en grunn til at vi ikke har oppgradert?

Reply:
Ikke annet enn at vi bruker biblioteker som er cross-kompilert (Blant annet våre egne). 
Stemmer for å ta det i en egen oppgave.

Reply:
Enig